shared/actions/teams.js

```diff
@@ -680,11 +694,7 @@ function* getTeamPublicity(_, action) {
     let tarsDisabled = false
     // can throw if you're not an admin
     try {
-      tarsDisabled = yield* Saga.callPromise(
-        RPCTypes.teamsGetTarsDisabledRpcPromise,
-        {name: teamname},
-        Constants.teamWaitingKey(teamname)
-      )
+      tarsDisabled = yield* Saga.callPromise(RPCTypes.teamsGetTarsDisabledRpcPromise, {name: teamname})
```

Comment:
why remove the waiting key here?

Reply:
This was a little tricky since this throws an error if you aren't an admin of the team. And reloadable will present itself if you have an error in one of your waiting keys. So it made it seem like the page had broken when in reality it was just a forgivable error. 

We can change this to use a different waiting key, but I just removed it since it didn't seem important to the user?

I can add another waiting key and have the progress indicator reference that one as well and have reloadable ignore it.

Reply:
my theory of the reload was the waiting keys it uses should be ones that if they fail mean the screen isn't correct and should be covered. so i think the waiting keys should be segregated.
so effectively you'd have a larger set of waiting keys for your waitingbuttons (tied to the actions/sagas) then a subset that are mission critical that are tied to the reload state

Reply:
sounds good